Makefile

```diff
@@ -175,15 +175,17 @@ target/assembly-$(assembly/VERSION)-word-addin : src/main/jre/OpenJDK11U-jdk_x64
 # gem install rubyzip
 src/main/jre/OpenJDK11U-jdk_x64_windows_hotspot_11.0.13_8/jdk-11.0.13+8 \
 src/main/jre/OpenJDK11U-jdk_x86-32_windows_hotspot_11.0.13_8/jdk-11.0.13+8 : %/jdk-11.0.13+8 : %.zip
-	require 'fileutils';                             \
-	require 'zip';                                   \
-	Zip::File.open("$<") { |zip_file|                \
-		zip_file.each { |f|                          \
-			f_path=File.join("$(dir $@)", f.name);   \
-			FileUtils.mkdir_p(File.dirname(f_path)); \
-			zip_file.extract(f, f_path);             \
-		}                                            \
-	}
+	require 'fileutils';                            	 \
+	require 'zip';                                  	 \
+	if !Dir.exist?("$<"[0..-5]);						 \
```

Comment:
What is the `[0..-5]` for?

I think there should be a more elegant way to deal with this. Why not delete the directory before the extraction happens? In that case the recipe should not be triggered unless the zip file is newer.

Reply:
See my latest commit.